python/pyarrow/_flight.pyx

```diff
@@ -887,6 +933,23 @@ cdef class FlightInfo(_Weakrefable):
         """The size in bytes of the data in this flight, or -1 if unknown."""
         return self.info.get().total_bytes()
 
+    @property
+    def ordered(self):
+        """Whether endpoints are in the same order as the data."""
+        return self.info.get().ordered()
+
+    @property
+    def app_metadata(self):
+        """
+        Application-defined opaque metadata.
+
+        There is no inherent or required relationship between this and the app_metadata fields in the FlightEndpoints
+        or resulting FlightData messages. Since this metadata is application-defined, a given application could define
+        there to be a relationship, but there is none required by the spec.
```

Comment:
```suggestion
        There is no inherent or required relationship between this and the
        app_metadata fields in the FlightEndpoints or resulting FlightData
        messages. Since this metadata is application-defined, a given
        application could define there to be a relationship, but there is
        none required by the spec.
```